app/js/components/answers/Show.react.js

```diff
@@ -48,30 +54,45 @@ class AllAnswers extends React.Component {
 
    render () {
      let answer = this.props.answer || {};
+     let question = this.props.question || {};
      let user = answer.user || {};
      let current_user = AuthStore.getCurrentUser();
-     let answer_edit_btn, answer_delete_btn;
+     let answer_edit_btn, answer_delete_btn, answer_accept_btn, accepted_answer_ribbon, accepted_answer;
      let answer_date = new Date(answer.created_at);
      let answer_dom_id = `answer-${answer.id}`;
-     var text_to_copy = `http://${window.location.host + window.location.pathname}#${answer_dom_id}`;
+     
+     var answer_href = `http://${window.location.host + window.location.pathname}#${answer_dom_id}`;
      if (current_user.id == user.id) {
        answer_edit_btn = <a href="#" className="item" data-question-id={answer.question_id}  data-id={answer.id} onClick={this.editAnswer.bind(this)}>edit</a>
```

Comment:
:red_circle: A bind call in the render path creates a brand new function on every single render.
This should be refactored like so in the constructor like so
```javascript 
    constructor(props) {
      super(props);

      this.editAnswer = this.editAnswer.bind(this);
    }
editAnswer(){
// your code
}
render(){
      answer_edit_btn = <a href="#" className="item" data-question-id={answer.question_id}  data-id={answer.id} onClick={this.editAnswer}>edit</a>
}```